service/internal/auth/authn.go

```diff
@@ -452,3 +456,17 @@ func validateDPoP(accessToken jwt.Token, acessTokenRaw string, dpopInfo dpopInfo
 	}
 	return &dpopKey, nil
 }
+
+func (a Authentication) isPublicRoute(path string) func(string) bool {
+	return func(route string) bool {
+		pattern := fmt.Sprintf("^%s$", route)
+		pattern = strings.ReplaceAll(pattern, "*", ".*")
```

Comment:
The risk here is that `.*` with turn into `..*` right?

Reply:
This is removed in place of filepath.Match